### DISCUSSION TITLE
What's needed for pull request to be reviewed/processed?

### DISCUSSION THREAD

**Discussion by olexp** [Codeception/Codeception#6262] [Q&A] [answered]
My pull request https://github.com/Codeception/module-laminas/pull/17 is sitting for 3 weeks without any feedback. I see new release was made 3 days ago. Does anyone look at pull requests before release? Or something else is needed? I didn't find any coding guidelines for that repository.
Thanks!

**Comment by Naktibalda** [comment, marked as answer]
> What's needed for pull request to be reviewed/processed? 

A: Pinging me that I missed it :)

The truth is that Laminas module has no maintainer that understands Laminas framework. Probably never had one either.

It seems that ZF2 module was created as a one-off contribution back in 2013.
I used ZF1 for a long time, so I did some work on ZF1 module and, as a "Zend guy", I took over maintenance of ZF2 module too.
But I've never migrated any of my projects to ZF2/ZF3 and I have no intention to start using Laminas.

You are welcome to stick around and do more work on a module. If I don't respond for a week, please mention me in comment to raise my attention.
